docs/src/main/paradox/docs/releases/index.md

```diff
@@ -28,19 +28,27 @@ The latest stable release is **v1.10.0** released on **18.12.2023**.
 - The S3 support for files has been completely rewritten
 - The default Elasticsearch views now uses a new mapping and settings which improves the ability to search for resources
   using the listing endpoints.
-- Resolvers/storages/views can't be tagged anymore
-- The global SSE endpoint, the SSE endpoints for realms/acls/organizations and the SSE endpoint 
-  to fetch indexing errors have been removed
-- Fetch organizations and projects by their uuid is now removed
 
 ### Deprecations
 
+- Resolvers/storages/views can't be tagged anymore
```

Comment:
Deprecations are things that are about to be removed in the next iteration.
For those, it is done, that is why it is in the breaking changes

Reply:
Can you confirm that the doc as it stands is ok or did I misclassify some of them ?

Reply:
I think that they should go back in the breaking changes as they are removed operations

Reply:
ok done and also added a definition as discussed offline